src/libsync/theme.cpp

```diff
@@ -517,6 +540,14 @@ QIcon Theme::folderOfflineIcon(bool sysTray) const
 
 QColor Theme::wizardHeaderTitleColor() const
 {
+    if (isDarkColor(wizardHeaderBackgroundColor()) &&
+            isDarkColor({APPLICATION_WIZARD_HEADER_TITLE_COLOR})){
+        return QColor("#ffffff");
```

Comment:
@camilasan What about adding a result QColor variable at the very beginning, then assigning to it, and then returning it at the end? This can potentially make the code more readable and fix styling errors.